ext/src/ruby_api/module.rs

```diff
@@ -4,7 +4,7 @@ use magnus::{function, method, Error, Module as _, Object, RString};
 use wasmtime::Module as ModuleImpl;
 
 #[derive(Clone)]
-#[magnus::wrap(class = "Wasmtime::Module")]
+#[magnus::wrap(class = "Wasmtime::Module", size, free_immediatly)]
```

Comment:
Not related to the PR, but these methods can be safely derived. The typo is intentional (mis-spelled in magnus).